x/auth/client/cli/suite_test.go

```diff
@@ -7,12 +7,13 @@ import (
 	"strings"
 	"testing"
 
-	"cosmossdk.io/core/address"
 	"cosmossdk.io/math"
 	abci "github.com/cometbft/cometbft/abci/types"
 	rpcclientmock "github.com/cometbft/cometbft/rpc/client/mock"
 	"github.com/stretchr/testify/suite"
 
+	"cosmossdk.io/core/address"
```

Comment:
nit: should cosmossdk.io/* imports be grouped together?

Reply:
Not sure

Reply:
should be treated as a third party import, should be grouped with others

Reply:
Oops, I've had my editor grouping `github.com/cosmos/cosmos-sdk,cosmossdk.io` together. I'll update that.